experiments/ClimaEarth/user_io/plot_helper.jl

```diff
@@ -90,3 +92,66 @@ function Plots.contourf(
         clims = clims,
     )
 end
+
+function get_ne(grid::CC.Grids.SpectralElementGrid2D)
+    return grid.topology.mesh.ne
+end
+function get_ne(grid::CC.Grids.LevelGrid)
+    return get_ne(grid.full_grid.horizontal_grid)
+end
+function get_ne(grid::CC.Grids.ExtrudedFiniteDifferenceGrid)
+    return get_ne(grid.horizontal_grid)
+end
+
+function get_R(grid)#::CC.Grids.SpectralElementGrid2D)
+    return CC.Grids.global_geometry(grid).radius
+end
+
+function get_height(grid::CC.Grids.ExtrudedFiniteDifferenceGrid)
+    return grid.vertical_grid.topology.mesh.domain.coord_max.z
+end
+function get_height(grid)
+    return nothing # 2d case
+end
+
+"""
+    make_cpu_field(field::CC.Fields.Field)
+
+For a CPU field, return the field unchanged.
+For a GPU field, copy the field and its underlying space onto the CPU.
+
+Note that this function allocates a new space and field,
+and should only be used for debugging and testing.
+"""
+function make_cpu_field(field::CC.Fields.Field)
+    if parent(field) isa Array
+        return field
+    else
+        # Copy field onto cpu space
+        space = axes(field)
+        FT = CC.Spaces.undertype(space)
+        R = get_R(space.grid)
+        ne = get_ne(space.grid)
+        polynomial_degree = CC.Quadratures.polynomial_degree(CC.Spaces.quadrature_style(space.grid))
+        nz = CC.Spaces.nlevels(space)
+        height = get_height(space.grid)
+
+        cpu_comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.CPUSingleThreaded())
+        cpu_space = TestHelper.create_space(
+            FT,
+            comms_ctx = cpu_comms_ctx,
+            R = R,
+            ne = ne,
+            polynomial_degree = polynomial_degree,
+            nz = nz,
+            height = height,
+        )
+        cpu_field = CC.Fields.ones(cpu_space)
+
+        parent(cpu_field) .= Array(parent(field))
+        return cpu_field
+    end
+end
+
+make_cpu_field(arr::Array) = arr
+make_cpu_field(arr::CUDA.CuArray) = Adapt.adapt(Array, arr)
```

Comment:
If we didn't support converting CuArrays to Arrays, we could remove the Adapt and CUDA dependencies, but I think it's good to include this since sometimes we plot fields and sometimes we plot arrays

Maybe this should be renamed to `make_cpu_object` instead of `make_cpu_field`